docs/getting-started/cli/README.md

```diff
@@ -0,0 +1,32 @@
+---
+description: Install and run Subsembly
+---
+
+# Subsembly CLI
```

Comment:
I think that this page should be replaced with:
https://subsembly.gitbook.io/subsembly/getting-started/cli/subsembly-cli